### DISCUSSION TITLE
[Esame Ottobre 2023] Esercizio 5

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#378] [Esami - Bandiera]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/7eff3097-5316-4f48-903b-6d9599de7a25)
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/133a6093-1f0e-4ab6-b811-862fdf3db95e)

**Comment by cornflexxx** [comment]
(a) Il modo più facile per calcolare il polinomio caratteristico, ovvero il $det(A-\lambda I_n)$, è attraverso la riduzione a scala di A:

$$A-\lambda I_n=\begin{vmatrix}
1-\lambda &  1& \cdots  & 1 \\
 1&  1-\lambda & \cdots  &   1 \\
 \vdots & \vdots &\ddots  & 1   \\
1  & 1 &  \cdots&   1-\lambda
\end{vmatrix}\longrightarrow \begin{vmatrix}
-\lambda &  0& \cdots  &0& \lambda \\
 0&  -\lambda &   &\vdots&   \vdots \\
 \vdots &  &\ddots  &0& \lambda   \\
 0 & \cdots &0  &-\lambda& \lambda   \\
1 & 1 &  \cdots&\cdots&   1-\lambda
\end{vmatrix} $$

La matrice di destra è ottenuta sottraendo l'ultima riga a tutte le altre $n-1$ righe. Per annullare l'ultima riga faccio $[A_n-\sum\frac{1}{\lambda}A_i]$

Quindi la matrice risultante sarà:

$$S=\begin{vmatrix}
-\lambda &  0& \cdots  &0& \lambda \\
 0&  -\lambda &   &&   \vdots \\
 \vdots &  &\ddots  && \lambda   \\
 \vdots &  & &-\lambda& \lambda   \\
0& \cdots&  \cdots&0&   n-\lambda
\end{vmatrix} $$

Essendo il $S$ una matrice triangolare superiore il suo determinante , e quindi polinomio caratteristico , sarà:

$$det(S)=(-1)^0\prod_{i=1}^n a_{ii}=(-\lambda)^{n-1}\cdot(n-\lambda)$$

Avremo quindi due radici , $\lambda=0$ con molteplicità algebrica $n-1$ e $\lambda=n$ con molteplicità $1$.

-----------------------
(b) Affinché una matrice sia diagonalizzabile le molteplicità algebriche di ciascun autovalore devono essere uguali a quelle geometriche, quindi  dobbiamo calcolare le molteplicità geometriche date dalla dimensione degli autospazi di ciascun autovalore. 
Quindi per il th. della  dimensione sappiamo che $dim Ker(A-\lambda I_n)=V_{\lambda}=dim R^n- dimIm(A-\lambda I_n)$
Sostituiamo $\lambda$ con le radici trovate prima : 
Chiamiamo $\alpha,\alpha'$ rispettivamente le molteplicità algebriche di $0$ e $n$.

$$rg(A-0I_n)=1\qquad\text{Infatti otteniamo tutte colonne dipendenti}$$

$$dim V_0=n-1=\alpha$$

--------------------

$$rg(A-nI_n)=rg\begin{vmatrix}
-n &  0& \cdots  &0& n \\
 0&  -n &   &\vdots&   \vdots \\
 \vdots &  &\ddots  &0& n   \\
 0 & \cdots &0  &-n& n   \\
1 & 1 &  \cdots&\cdots&   1-n
\end{vmatrix}=rg\begin{vmatrix}
-n &  0& \cdots  &0& n \\
 0&  -n &   &\vdots&   \vdots \\
 \vdots &  &\ddots  &0& n   \\
 0 & \cdots &0  &-n& n   \\
0 & 0 &  \cdots&\cdots&  0
\end{vmatrix}=n-1$$

Semplificando con gauss otteniamo una matrice strutturalmente identica a quella del punto precedente, in questo caso però possiamo annullare totalmente l'ultima riga svolgendo lo stesso calcolo precedente $[A_n-\sum\frac{1}{n}A_i]$, quindi ci riconduciamo ad una matrice a scala con $rg=n-1$, ne segue:

$$dim V_n=n-(n-1)=1=\alpha'$$

Allora la matrice $A$ è diagonalizzabile.

--------------

Allora $V_0$ è uno spazio generato dallo span dei seguenti vettori:

$$V_0=Ker A=Span\begin{pmatrix}\begin{vmatrix}
-1\\
1\\
0\\
0\\
\vdots\\
0
\end{vmatrix},\begin{vmatrix}
-1\\
0\\
1\\
0\\
\vdots\\
0
\end{vmatrix},\cdots,\begin{vmatrix}
-1\\
0\\
0\\
0\\
\vdots\\
1
\end{vmatrix}\end{pmatrix}\qquad \text{In totale $n-1$ vettori l.i.}$$

Allo stesso modo calcolo l'unico generatore di $V_n$:

$$V_n=Span\begin{pmatrix}\begin{vmatrix}
1\\
1\\
\vdots\\
1
\end{vmatrix}\end{pmatrix}$$

Quindi l'unione delle basi degli autospazi costituisce una base di $\mathbb{R}^n$, quindi la matrice $B$ invertibile è la matrice di cambio di base da quella di autovettori a quella canonica , e la matrice $D$ , diagonale, è quella che ha sulla diagonale gli autovalori ordinati per autovettori : 

$$\mathcal{M}_{\mathcal{B}\mathcal{B}}(A)=D=\begin{vmatrix}
0&\cdots&\cdots&0\\
\vdots&\ddots&&\vdots\\
\vdots&&\ddots&0\\
0&\cdots&0&n
\end{vmatrix}$$

$$\mathcal{M}_{\mathcal{E}\mathcal{B}}(Id_n)=B=\begin{vmatrix}
-1&-1&\cdots &\cdots&1\\
1&0& &&1\\
0&1&  &  & \vdots\\
0&0& & & \vdots\\
\vdots & \vdots &  & & \vdots\\
0 & 0 & \cdots & \cdots&1
\end{vmatrix}$$
